GTBitsOfGood/national-npp
# Samrat lyann/42 nonprofit profile

## What is this?
This pull request includes:
- User interface changes for the nonprofit profile to match Figma designs
- Prepopulating + submitting data when editing the nonprofit profile
- Updated mongoose models to match the new pages
- Error notifications where needed


## Setup



## Steps to Test


- In MongoDB, make sure your user profile has an associated nonprofit attached to it
- In the roles give yourself either "Nonprofit Admin" or "Nonprofit Member"
- Navigate to http://localhost:3000/nonprofit/profile
- Test whether admins can change info and whether this info updates in MongoDB
- Test whether members can view info but not change info
- Test whether required fields work as intended by emptying a required field and pressing "Save Changes"

## Associated Issues

- #42



## Additional Instructions

- We did not implement the phone number validation because Matt said he would create a standard function across both profiles
